pykern/pkcli/github.py

```diff
@@ -28,12 +28,14 @@
 _GITHUB_HOST = "github.com"
 _GITHUB_URI = "https://" + _GITHUB_HOST
 _GITHUB_API = "https://api." + _GITHUB_HOST
-_WIKI_ERROR_OK = (
-    r"fatal: remote error: access denied or repository not exported: .*wiki.git"
-)
+_WIKI_ERRORS_OK = [
```

Comment:
Agreed

I was seeing both `remote: Repository not found.` and `fatal: repository 'https://github.com/radiasoft/.*.wiki.git/' not found` errors while testing because I didn't cast `e.output` to a string before printing.